### PR TITLE
German Translation for the 2024 annual survey

### PR DESCRIPTION
**Pull request by llogiq**
r? @Kobzol

### CODE REVIEW

File: surveys/2024-annual-survey/translations/de.md [hunk 1]
```diff
@@ -183,8 +183,8 @@ Type: select one
 - 1.79
 - 1.78
 - 1.77
-- 1.76
-- 1.75
+- 1,76
+- 1,75
```
Comment (jieyouxu) [inline surveys/2024-annual-survey/translations/de.md:187]:
```suggestion
- 1.76
- 1.75
```

(version numbers are separator-sensitive, probably can't be localized)

Comment (apiraino) [reply surveys/2024-annual-survey/translations/de.md:187]: All commas in rustc version numbers have been fixed in https://github.com/rust-lang/surveys/commit/945c1edb8a2c6d55e8d619dba6c7613ac94bd258

Comment (llogiq) [reply surveys/2024-annual-survey/translations/de.md:187]: Oops. Typo, sorry.

**Review by apiraino — APPROVED** [review]

File: surveys/2024-annual-survey/translations/de.md (hunk 1 above)
Comment (Kobzol) [reply surveys/2024-annual-survey/translations/de.md]: I fixed the two commas in SurveyHero and reverted them from this PR.

**Comment by Kobzol** [comment]
We haven't received further feedback on this translation, so let's merge it.

**Comment by Kobzol** [comment]
Thank you for the translation!

**Comment by llogiq** [comment]
My pleasure.

**Pull request merged**